prime-router/src/main/kotlin/USTimeZone.kt

```diff
@@ -12,4 +13,6 @@ enum class USTimeZone(val zoneId: String) {
     INDIANA_STARKE("US/Indiana-Starke"),
     MICHIGAN("US/Michigan"),
     CHAMORRO("Pacific/Guam"),
+    // not technically a US time zone but we need it
```

Comment:
Is is conceivable that we get results from a non-US timezone? Should we error if we do?